client/v1/status.go

```diff
@@ -0,0 +1,46 @@
+package v1
+
+import (
```

Comment:
we should rename the file name, such as packages.go? or package_status.go? `status.go` is too general.